eclair-core/src/main/scala/fr/acinq/eclair/channel/fsm/ChannelOpenDualFunded.scala

```diff
@@ -18,7 +18,7 @@ package fr.acinq.eclair.channel.fsm
 
 import akka.actor.typed.scaladsl.adapter.{ClassicActorContextOps, actorRefAdapter}
 import akka.actor.{ActorRef, Status}
-import com.softwaremill.quicklens.{ModifyPimp, QuicklensAt}
+import com.softwaremill.quicklens.{ModifyPimp, QuicklensAt, QuicklensWhen}
```

Comment:
nit: this is unused

Reply:
Done in [04cd2d9](https://github.com/ACINQ/eclair/pull/2592/commits/04cd2d988e59c712ad535c3ddd5287ed0ad9f688).